torch/csrc/autograd/python_engine.cpp

```diff
@@ -420,6 +416,17 @@ PyObject* THPEngine_queue_callback(PyObject* self, PyObject* _callback) {
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPEngine__exec_final_callbacks_stub(
```

Comment:
We're mainly using this stub to guide dynamo to the right variable easier, I think instead we can create a custom class e.g. CompiledAutogradEngine in external_utils, similar to how we used to route FakeContext